dora/underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystemFactory.java

```diff
@@ -46,16 +47,16 @@ public GCSUnderFileSystemFactory() {}
   public UnderFileSystem create(String path, UnderFileSystemConfiguration conf) {
     Preconditions.checkNotNull(path, "Unable to create UnderFileSystem instance:"
         + " URI path should not be null");
-
-    if (checkGCSCredentials(conf)) {
-      if (conf.getInt(PropertyKey.UNDERFS_GCS_VERSION) == GCS_VERSION_TWO) {
-        try {
-          return GCSV2UnderFileSystem.createInstance(new AlluxioURI(path), conf);
-        } catch (IOException e) {
-          LOG.error("Failed to create GCSV2UnderFileSystem.", e);
-          throw Throwables.propagate(e);
-        }
-      } else {
+    if (conf.getInt(PropertyKey.UNDERFS_GCS_VERSION) == GCS_VERSION_TWO) {
+      try {
+        return GCSV2UnderFileSystem.createInstance(new AlluxioURI(path), conf);
```

Comment:
why checkgcscredentials doesnt apply to this anymore?

Reply:
GCS v2 only need to provide json file, only v1 is using access key. #17985 make the wrong check so I just revert it

Reply:
then why it was merged in the first place? sorry I don't have the context and I don't want to make the call of LGTM, if some bugs are introduced as part of other PR, can u plz let the related group know?

Reply:
I think this change is OK for GCS v2.